reconcile/gql_definitions/terraform_cloudflare_users/terraform_cloudflare_roles.gql

```diff
@@ -0,0 +1,38 @@
+# qenerate: plugin=pydantic_v1
+
+query CloudflareAccountRole {
+  cloudflare_account_roles:	cloudflare_account_role_v1 {
+    name
+    roles
+    access_roles {
+      users {
+        cloudflare_user
+        org_username
```

Comment:
nit: Would it make sense to extend our `User` fragment with `cloudflare_user` and use it here?

```graphql
... User
```

Reply:
I chose to keep it as is for now since this integration is not utilizing other fields in `User` fragment.